circleci-provision-scripts/base.sh

```diff
@@ -47,10 +47,25 @@ EOF
     apt-get update -y
 
     # Install base packages
-    apt-get install \
-        build-essential cmake git-core mercurial zip gdb \
-        htop emacs vim nano lsof vnc4server tmux lzop \
-        build-essential htop emacs vim nano lsof tmux zip vnc4server \
-        curl unzip git-core ack-grep software-properties-common build-essential \
-        parallel libav-tools make autoconf liblapack-dev gfortran
+    cat << EOS | xargs apt-get install
+autoconf
+bsdtar
+build-essential
+cmake
+curl
+gfortran
+git-core
+libav-tools
+liblapack-dev
+lsof
+lzop
+make
+mercurial
+parallel
+software-properties-common
+tmux
+unzip
+vim
+zip
+EOS
```

Comment:
`tmux`, `lsof` belong in the devtools as well.  I would move `gfortran` and `liblapack-dev` to its their own functions - Don't think we have many fortran projects :).

I'm torn about `build-essential`, `autoconf`, `cmake`, `make` .  They are primarily relevant to C projects - but other subsequent modules (e.g. ruby, php).  I probably would remove it from base.sh to its own module (e.g. base compilers).